### PR TITLE
Fix Github OAuth not working with private Organizations

### PR DESCRIPTION
**Pull request by lostick**
Fix for the organizations endpoint to allow fetching private and public orgs.

### CODE REVIEW

File: conf/defaults.ini [hunk 1]
```diff
@@ -248,7 +248,7 @@ enabled = false
 allow_sign_up = true
 client_id = some_id
 client_secret = some_secret
-scopes = user:email
+scopes = user:email,read:org
```
Comment (torkelo) [inline conf/defaults.ini]: you need to change sample.ini as well.

**Review by torkelo — COMMENTED** [review]
have you tested with public and private orgs, this looks similar to https://github.com/grafana/grafana/pull/10851/files that I had to revert as it broke it for public orgs

File: pkg/social/github_oauth.go [hunk 2]
```diff
@@ -195,10 +195,9 @@ func (s *SocialGithub) FetchOrganizations(client *http.Client, organizationsUrl
 func (s *SocialGithub) UserInfo(client *http.Client, token *oauth2.Token) (*BasicUserInfo, error) {
 
 	var data struct {
-		Id               int    `json:"id"`
-		Login            string `json:"login"`
-		Email            string `json:"email"`
-		OrganizationsUrl string `json:"organizations_url"`
```
Comment (torkelo) [inline pkg/social/github_oauth.go]: the the organizations_url property github returns not correct for private orgs?

File: conf/defaults.ini (hunk 1 above)
Comment (DanCech) [reply conf/defaults.ini]: scope values are usually separated with a space character, not a comma

Comment (lostick) [reply conf/defaults.ini]: @DanCech github normalizes them as comma separated, if i'm not mistaken.

Comment (lostick) [reply conf/defaults.ini]: @torkelo [sample.ini](https://github.com/grafana/grafana/blob/master/conf/sample.ini#L235) seems fine

Comment (DanCech) [reply conf/defaults.ini]: https://developer.github.com/apps/building-oauth-apps/scopes-for-oauth-apps/ read the note

Comment (lostick) [reply conf/defaults.ini]: @DanCech sure, I can use space delimited scopes if needed.  
`SplitString` function handles both cases anyway 👍

File: pkg/social/github_oauth.go (hunk 2 above)
Comment (lostick) [reply pkg/social/github_oauth.go]: @torkelo indeed, that api endpoint won't list all repos.

**Comment by lostick** [comment]
@torkelo yep, compiled and tested with private and public repos.

**Pull request merged**